dist/constants.json

```diff
@@ -6923,899 +6923,899 @@
       {
         "value": "674A4",
-        "label": "WACO, TX VAMC"
+        "label": "Doris Miller Department of Veterans Affairs Medical Center"
       },
       {
         "value": "674BY",
-        "label": "AUSTIN VA CLINIC"
+        "label": "Austin VA Clinic"
       },
       {
         "value": "580GD",
-        "label": "CONROE CBOC"
+        "label": "Conroe VA Clinic"
       },
       {
         "value": "580GE",
-        "label": "KATY CBOC"
+        "label": "Katy VA Clinic"
       },
       {
         "value": "580GH",
-        "label": "TOMBALL CBOC"
+        "label": "Tomball VA Clinic"
       },
       {
         "value": "671GP",
-        "label": "BALCONES HEIGHTS VA CLINIC"
+        "label": "Balcones Heights VA Clinic"
       },
       {
         "value": "740GH",
-        "label": "SOUTH ENTERPRIZE VA CLINIC"
+        "label": "South Enterprize VA Clinic"
       },
       {
         "value": "740GJ",
-        "label": "NORTH TENTH STREET VA CLINIC"
+        "label": "North Tenth Street VA Clinic"
       },
       {
         "value": "671GQ",
-        "label": "SHAVANO PARK VA CLINIC"
+        "label": "Shavano Park VA Clinic"
       },
       {
         "value": "674GF",
-        "label": "TEMPLE VA CLINIC"
+        "label": "Temple VA Clinic"
       },
       {
         "value": "740GI",
-        "label": "OLD BROWNSVILLE VA CLINIC"
+        "label": "Old Brownsville VA Clinic"
       },
       {
         "value": "549GM",
-        "label": "GRAND PRAIRIE VA CLINIC"
+        "label": "Grand Prairie VA Clinic"
       },
       {
         "value": "549GL",
-        "label": "PLANO VA CLINIC"
+        "label": "Plano VA Clinic"
       },
       {
         "value": "756QB",
-        "label": "EL PASO CENTRAL VA CLINIC"
+        "label": "El Paso Central VA Clinic"
       },
       {
         "value": "580QB",
-        "label": "HOUSTON VA MOBILE CLINIC"
+        "label": "Houston VA Mobile Clinic"
       },
       {
         "value": "549GK",
-        "label": "POLK STREET VA CLINIC"
+        "label": "Polk Street VA Clinic"
       },
       {
         "value": "580GJ",
-        "label": "TEXAS CITY VA CLINIC"
+        "label": "Texas City VA Clinic"
       },
       {
         "value": "504",
-        "label": "AMARILLO HCS"
+        "label": "Thomas E. Creek Department of Veterans Affairs Medical Center"
       },
       {
         "value": "519",
-        "label": "WEST TEXAS VA HEALTH CARE SYSTEM - BIG SPRING DIVISION"
+        "label": "George H. O'Brien, Jr., Department of Veterans Affairs Medical Center"
       },
       {
         "value": "671",
-        "label": "AUDIE L. MURPHY MEMORIAL HOSP"
+        "label": "Audie L. Murphy Memorial Veterans' Hospital"
       },
       {
         "value": "674",
-        "label": "OLIN E. TEAGUE VET CENTER"
+        "label": "Olin E. Teague Veterans' Center"
       },
       {
         "value": "756",
-        "label": "EL PASO HCS"
+        "label": "El Paso VA Clinic"
```

Comment:
Flagging this label since it goes from an entire health care system to a single facility.